QWeb/internal/browser/chrome.py

```diff
@@ -127,7 +126,10 @@ def create_chrome_options(chrome_args: Optional[list[str]], **kwargs: Any) -> Op
 
 def resolve_chromedriver_path(executable_path: str) -> str:
     """Resolve the path to the ChromeDriver executable."""
-    return util.get_rfw_variable_value("${CHROMEDRIVER_PATH}") or executable_path
+    driver_path = util.get_rfw_variable_value("${CHROMEDRIVER_PATH}") or executable_path
+    if driver_path:
+        logger.debug(f"Chromedriver path: {driver_path}")
```

Comment:
Setting path with `executable_path` should be intentional by user and IMO not necessary to log.
But the override with with rf variable is a 'global' side-effect and context about the change should be provided in the log message.

```
rf_path = util.get_rfw_variable_value("${CHROMEDRIVER_PATH}")
driver_path = rf_path or executable_path
if driver_path and rf_path:
        logger.debug(f"Setting webdriver executable path with Robot Framework variable ${{CHROMEDRIVER_PATH}} to ${driver_path}")
```

_internal/browser/edge.py_ has similar override with `${EDGEDRIVER_PATH}` and that override should also be logged in this fashion

Reply:
nope, sorry this is how it will be. This will log the one that will be used, regardless which one was intentionally or unintentionally changed.

Reply:
edgedriver custom path logging added